install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -553,6 +553,62 @@ INSERT INTO `PREFIX_hook_alias` (`name`, `alias`) VALUES
 ALTER TABLE `PREFIX_order_detail` ADD `total_refunded_tax_excl` DECIMAL(20, 6) NOT NULL DEFAULT '0.000000' AFTER `original_wholesale_price`;
 ALTER TABLE `PREFIX_order_detail` ADD `total_refunded_tax_incl` DECIMAL(20, 6) NOT NULL DEFAULT '0.000000' AFTER `total_refunded_tax_excl`;
 
+ALTER TABLE `PREFIX_group_reduction` CHANGE `reduction` `reduction` DECIMAL(5, 4) NOT NULL DEFAULT '0.0000';
+ALTER TABLE `PREFIX_product_group_reduction_cache` CHANGE `reduction` `reduction` DECIMAL(5, 4) NOT NULL DEFAULT '0.0000';
+ALTER TABLE `PREFIX_order_slip` CHANGE `amount` `amount` DECIMAL(20, 6) NOT NULL DEFAULT '0.000000';
+ALTER TABLE `PREFIX_order_slip` CHANGE `shipping_cost_amount` `shipping_cost_amount` DECIMAL(20, 6) NOT NULL DEFAULT '0.000000';
+ALTER TABLE `PREFIX_order_payment` CHANGE `amount` `amount` DECIMAL(20, 6) NOT NULL DEFAULT '0.000000';
+
+/* attribute_impact price */
+ALTER TABLE `PREFIX_attribute_impact` ADD `price_tmp` DECIMAL(20, 6) NOT NULL DEFAULT '0.000000' AFTER `price`;
+UPDATE `PREFIX_attribute_impact` SET `price_tmp` = `price`;
```

Comment:
You can't do it:
```
#1264 - Out of range value for column 'price_tmp' at row 1
```

Reply:
@PierreRambaud I don't see why? I've tried it and didn't have any problem 🤔

Reply:
You need to insert the max value in the original price field.
After you can set `price_tmp` with an higher value than it can received.

You need to do something like
```sql
UPDATE `ps_attribute_impact` SET price_tmp = CASE WHEN LENGTH(price) < 18 
    THEN price
    ELSE RIGHT(price, 17)
END
```
Which will have:
![image](https://user-images.githubusercontent.com/1462701/80471179-56c84100-8943-11ea-8338-3ea7251947e1.png)

You don't need to create a temporary column.
If it doesn't fail for you, you need to check your MySQL / MariaDB version :thinking: 

I'm using 
![image](https://user-images.githubusercontent.com/1462701/80471329-8aa36680-8943-11ea-9948-a2e2525f1d8a.png)

and your script is prompt errors.